core/proto_test.go

```diff
@@ -110,6 +124,22 @@ func TestSetBlockSig(t *testing.T) {
 	require.NotEqual(t, clone.Signature(), block.Signature())
 }
 
+func TestSetBlindedBlockSig(t *testing.T) {
```

Comment:
may be wrong but this isn't testing anything in the proto.go file as far as i can see

should it be included it's copied from the TestSetBlockSig

Reply:
yeah, these tests should move to `signeddata_test` since it tests whether `SignedData` implementation is immutable and that `SetSignature` returns a new instance.